application-favorites-default/src/main/java/org/xwiki/contrib/favorites/internal/filter/output/FavoriteInstanceOutputProperties.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package org.xwiki.contrib.favorites.internal.filter.output;
+
+import org.xwiki.filter.instance.output.InstanceOutputProperties;
+
+/**
+ * Favorites Instance Output Properties.
+ * @since $1.4.0$
+ * @version $Id$
+ */
+public class FavoriteInstanceOutputProperties extends InstanceOutputProperties
```

Comment:
Just in case you think about stuff to add to it later ?

Reply:
I was just wondering if you really need that custom FavoriteInstanceOutputProperties. If you are wondering if you can introduce this later when really needed without breaking anything, yes I think so.